themes/digital.gov/layouts/communities/single.html

```diff
@@ -33,139 +32,128 @@ <h1>{{ .Title | markdownify }} Community</h1>
     <section>
       <div class="grid-container grid-container-desktop">
         <div class="grid-row tablet-lg:grid-gap-4">
-          <div class="grid-col-12 tablet-lg:grid-col-4">
 
-            {{- if .Params.community_list -}}
-            <section class="community-info {{ .Params.community_type -}}">
-
-              {{- range $i, $e := .Params.community_list -}}
-                <div class="join-box">
+          <div class="grid-col-12 tablet-lg:grid-col-8">
 
-                  <div class="type">
-                    {{/*
-                      Checks to see if the community is public or govt only and sets the "terms" of the community, with an option to override the terms for each community by setting it in the front matter
-                    */}}
+            {{/* Summary */}}
+            {{- if .Params.summary -}}
+            <div class="content">
+              <p>{{- .Params.summary | markdownify -}}</p>
+
+              {{- if .Params.community_list -}}
+              <section class="join-community-form">
+                <h4 class="join-community-form__header">
+                  <svg class="join-community-form__header-svg usa-icon dg-icon dg-icon--extra-large margin-bottom-05" aria-hidden="true" focusable="false">
+                    <use xlink:href="{{ "uswds/img/sprite.svg#groups" | relURL }}"></use>
+                  </svg> Connect with us
```

Comment:
The text should be on it's own line for clarity.

Reply:
Moved to new line